timemachine/cpp/src/barostat.hpp

```diff
@@ -19,7 +19,8 @@ template <typename RealType> class MonteCarloBarostat {
         std::vector<std::vector<int>> group_idxs,
         const int interval,
         std::vector<std::shared_ptr<BoundPotential>> bps,
-        const int seed);
+        const int seed,
+        const bool adapt_volume_scale_factor);
```

Comment:
nit: generally it seems like we should be consistent between getter/setter names, names of the setter argument, and the name of the private variable that's being updated

Reply:
Unified in [b102405](https://github.com/proteneer/timemachine/pull/1152/commits/b10240596853c2ab57047e8b153a6536fe373412)